### ISSUE TITLE
Redcarpet incorrectly parses nested lists

### ISSUE THREAD

**Issue by kristoph** [vmg/redcarpet#790]
5. Lists: You can create a bulleted list by prefacing each line with a dash or an asterisk. For example:

   * Item 1
   * Item 2   
   * Item 3
   
( or equivalent string "5. Lists: You can create a bulleted list by prefacing each line with a dash or an asterisk. For example:\n\n   * Item 1\n   * Item 2\n   * Item 3" )

Generates ...

"<ol>\n<li>Lists: You can create a bulleted list by prefacing each line with a dash or an asterisk. For example:</li>\n</ol>\n\n<ul>\n<li>Item 1</li>\n<li>Item 2</li>\n<li>Item 3</li>\n</ul>\n"

**Comment by jlpereira** [comment]
Any news on this? We are also having this problem with nested lists

**Comment by dhMuse** [comment]
Nope, sorry, the point in my last post about the surrounding `<li>` tags being stripped was mistaken! Passing in correctly nested `<ul><li><ol></ol></li></ul>` will retain the `<li>` tags.
